airflow/models/mappedoperator.py

```diff
@@ -57,6 +60,13 @@ def expand_start_from_trigger(self, *, context: Context, session: Session) -> bo
 
         :meta private:
         """
+        if self.partial_kwargs.get("start_from_trigger", self.start_from_trigger):
+            log.warning(
+                "Starting a mapped task from triggerer is currently unsupported",
+                task_id=self.task_id,
+                dag_id=self.dag_id,
+            )
+        return False
```

Comment:
Should this be warning or exception?
If user is doing `TimeSensorAsync.partial(..., start_from_trigger=True).expand(...)` wouldn't it be better to show broken dag message with noting that start_from_trigger must be false in this case

Reply:
Cool
please consider not closing the linked issue but only to reduce it's priority from critical?
We better to close the issue only when we have the broken dag message in place.

Reply:
ok